baybe/searchspace/discrete.py

```diff
@@ -130,33 +128,9 @@ def __str__(self) -> str:
         param_df = pd.DataFrame(param_list)
         constraints_df = pd.DataFrame(constraints_list)
 
-        # Get summary information from metadata
-        was_recommended_count = len(self.metadata[self.metadata[_METADATA_COLUMNS[0]]])
-        was_measured_count = len(self.metadata[self.metadata[_METADATA_COLUMNS[1]]])
-        dont_recommend_count = len(self.metadata[self.metadata[_METADATA_COLUMNS[2]]])
-        metadata_count = len(self.metadata)
-
-        metadata_fields = [
```

Comment:
this should be moved to campaign and not fully removed

Reply:
Agree, I simply forgot to bring it back after the migration, but it's there now. Please check and resolve if OK